pkg/cmd/extension/command.go

```diff
@@ -219,6 +221,33 @@ func NewCmdExtension(f *cmdutil.Factory) *cobra.Command {
 				return nil
 			},
 		},
+		&cobra.Command{
+			Use:   "run <name> [args]",
+			Short: "Run an installed extension",
+			Long: heredoc.Doc(`
+			Run an extension using either the extension name, or the extension repository's
```

Comment:
Nit: we usually indent the contents of a heredoc one level below the line that starts the heredoc

Reply:
A few others in command.go weren't so I fixed those as well.